django_migration_linter/sql_analyser/analyser.py

```diff
@@ -37,7 +40,12 @@ def get_sql_analyser_class_from_db_vendor(database_vendor):
     elif "sqlite" in database_vendor:
         sql_analyser_class = SqliteAnalyser
     else:
-        raise ValueError("Unsupported database vendor. Try specifying an SQL analyser.")
+        raise ValueError(
+            "Unsupported database vendor '{}'. "
+            "Specify 'sql_analyser' in MIGRATION_LINTER_OPTIONS.".format(
+                database_vendor
```

Comment:
There are multiple ways to specify this. For me it made the most sense to specify it in the settings (`MIGRATION_LINTER_OPTIONS`) where all the other database options live. If you feel like this is too specific, I can also leave out this part.

Important for me was to mention `sql_analyser` at least, since `Try specifying an SQL analyser` was too broad for me and I had no clue how to do this, leading to stepping through the libraries source code.